draft-ietf-mls-protocol.md

```diff
@@ -4744,7 +4744,10 @@ to an adversary solely by the ciphertext length.
 The length of the `padding` field in `MLSCiphertextContent` can be
 chosen at the time of message encryption by the sender. Senders may use padding
 to reduce the ability of attackers outside the group to infer the size of the
-encrypted content.
+encrypted content.  Note, however, that the transports used to carry MLS
+messages may have maximum message sizes, so padding schemes SHOULD avoid
+increasing message size beyond these any such limits that exist in a given
```

Comment:
Personally, I think it is necessary that the the resulting IP packets do not exceed the actual PMTU. This will "black hole" the data, the network will drop these packets. So, saying "SHOULD" kind of looks like there might be a way around that exists - whereas really the way around is to make sure this does not happen.

 I wonder in this case where it might be better to say "needs to avoid sending IP packets that are larger than this size" - this is of course often handled by lower protocol layers, and you may wish to also note that